src/material-experimental/mdc-theming/_all-theme.scss

```diff
@@ -27,6 +28,7 @@
     @include mat-mdc-core-theme($theme-or-color-config);
     @include mat-mdc-autocomplete-theme($theme-or-color-config);
     @include mat-mdc-button-theme($theme-or-color-config);
+    @include mat-mdc-button-toggle-theme($theme-or-color-config);
```

Comment:
did you verify that this is applying the typography styles as well?

Reply:
Yes when I removed the typography styles from the theme file the text looked different